features/step_definitions/networking.rb

```diff
@@ -591,7 +591,7 @@
   available_multus_replicas = daemon_set('multus', project('openshift-multus')).replica_counters(user: admin)[:available]
   #storing desired_multus_replicas value in desired_multus_replicas clipboard variable as well
   cb.desired_multus_replicas = desired_multus_replicas
-  unless desired_multus_replicas == available_multus_replicas && available_multus_replicas != 0
+  unless (desired_multus_replicas == available_multus_replicas || desired_multus_replicas >= env.nodes.count) && available_multus_replicas != 0
```

Comment:
I am not sure in manual testing whether need desired_multus_replicas == available_multus_replicas .   If only need some available_multus_replicas, I think mybe we just simply use "available_multus_replicas >= $num"  and select available nodes to deploy pods.

For this "|| desired_multus_replicas >= env.nodes.count",  if some nodes are in not ready status,  also meet this requirement, but may got the pods deployed failed on the NotReady node.

Reply:
> I am not sure in manual testing whether need desired_multus_replicas == available_multus_replicas . If only need some available_multus_replicas, I think mybe we just simply use "available_multus_replicas >= $num" and select available nodes to deploy pods.

I guess the original design purpose is make sure all of multus pods are running well in cluster so here use desired_multus_replicas == available_multus_replicas 
yes, IMO, multus test case should be able to continue to run even if there is one multus pod of cluster is not working well. 
However if use `"available_multus_replicas >= $num"`, if the number is 3,  in case there is one multus pod of one node is not working well and the following testpod was scheduled that node. I guess we will also get unexpected result. 

> For this "|| desired_multus_replicas >= env.nodes.count", if some nodes are in not ready status, also meet this requirement, but may got the pods deployed failed on the NotReady node.

here should use `>` instead of `>=`

Reply:
@zhaozhanqi If env.nodes.count only return the num of ready nodes, how about just check  if "available_multus_replicas == env.nodes.count" to make sure multus pods are running well in all available nodes? It will be a multus bug if this check fail.
Thanks for quick solution on this issue.

Reply:
I agree with @weliang1 I think it makes sense.  If you have multus running on all the nodes in "Ready" state that might suffice. Hence `available_multus_replicas == env.nodes.count`

Reply:
yes, my first version want to use that, However after I found multus pod only scheduled on 'linux', see below,  So if the cluster have one 'windows' worker.  that's mean available_multus_replicas will < env.nodes.count
```
multus                        5         5         5       5            5           kubernetes.io/os=linux 
```

Reply:
would it make sense to say `available_multus_replicas will <= env.nodes.count`

Reply:
Seems like:
If we have scaling up node case running in parallel, then desired_multus_replicas > env.nodes.count,
If we have a window node running in the cluster, then desired_multus_replicas <  env.nodes.count,

If we can get env.linux_nodes.count , shoul “available_multus_replicas == env.linux_nodes.count” pass in both scale up node and window cluster?

Reply:
> would it make sense to say `available_multus_replicas will <= env.nodes.count` ??

`<=` should not be good, since if the `available_multus_replicas` is 0 <= 5,  we cannot make sure the cluster is normal

Reply:
> Seems like:
> If we have scaling up node case running in parallel, then desired_multus_replicas > env.nodes.count,
scaling up node case should be running in queue
`desired_multus_replicas > env.nodes.count` only happen in short time. means when the node number restore to original number. but `desired_multus_replicas` has not been restored.   However now our test case begin to run thus caused this issue. 

So I think this `desired_multus_replicas == available_multus_replicas || desired_multus_replicas > env.nodes.count` can make sure the cluster is normal one at least.


> If we have a window node running in the cluster, then desired_multus_replicas < env.nodes.count,
> 
> If we can get env.linux_nodes.count , shoul “available_multus_replicas == env.linux_nodes.count” pass in both scale up node and window cluster?

I think this `desired_multus_replicas == available_multus_replicas || desired_multus_replicas > env.nodes.count` can make sure the cluster is normal one.

Reply:
/LGTM